src/BizHawk.Emulation.Cores/Consoles/GCE/Vectrex/PPU.cs

```diff
@@ -40,8 +40,8 @@ public void tick()
 			{
 				if (skip == 0)
 				{
-					x_pos = x_pos + (x_vel - 128.0) / 256.0 * (vec_scale + 2);
-					y_pos = y_pos - (y_vel - 128.0) / 256.0 * (vec_scale + 2);
+					x_pos += (x_vel - 128.0) / 256.0 * (vec_scale + 2);
+					y_pos -= (y_vel - 128.0) / 256.0 * (vec_scale + 2);
```

Comment:
the operation on these positions could be DRYed by abstracting out to a function

Reply:
This and the things below may be valid, but I don't really wanna refactor the code here, just apply basic style rules.